src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CatalogueSolutionsController.cs

```diff
@@ -88,14 +88,18 @@ public async Task<ActionResult<GetCatalogueSolutionOrderItemModel>> GetOrderItem
             if (orderItem is null)
                 return NotFound();
 
+            var serviceRecipients = order.ServiceRecipients;
+
             return new GetCatalogueSolutionOrderItemModel
             {
                 ServiceRecipient = new ServiceRecipientModel
                 {
-                    OdsCode = orderItem.OdsCode
+                    OdsCode = orderItem.OdsCode,
+                    Name = serviceRecipients.FirstOrDefault(serviceRecipient => string.Equals(orderItem.OdsCode,
+                        serviceRecipient.OdsCode, StringComparison.Ordinal))?.Name
```

Comment:
I think this should be InvariantCulture or OrdinalIgnoreCase, not just Ordinal. Ordinal will treat case sensitivity differently... is that what we want?

Reply:
This is using the same ones as other, so might be a tech debt issue, as we should then change the others ect. Or I can do it in my next PR, as I'm doing another service recipient name then